Library/Homebrew/cmd/update-report.rb

```diff
@@ -302,16 +302,11 @@ def migrate_legacy_repository_if_necessary
     $stderr.puts e.backtrace
   end
 
-  def link_completions_and_docs(repository = HOMEBREW_REPOSITORY)
+  def link_additional_files(repository = HOMEBREW_REPOSITORY)
```

Comment:
1) YAGNI, 2) we can change the name then. `link_additional_files` doesn't really communicate anything.